src/dist/static/index.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const settings = allure.getPluginSettings('playwright-traceview-plugin', {default_trace_app_url: "https://playwright.aqa.forte.io"});
```

Comment:
Can you change the URL to point to the playwright's own instance of trace viewer